File: global/nextjs/src/utils/api/templates/Task.ts
export class Task{
    private nome: string;

    private concluida: boolean;

    constructor(nome: string, concluida?: boolean) {
        this.nome = nome;
        this.concluida = concluida ? concluida : false;
    }

    public getNome(): string
    {
        return this.nome;
    }

    public setNome(nome: string): void
    {
        this.nome = nome;
    }

    public getConcluida(): boolean
    {
        return this.concluida;
    }

    public setConcluida(concluida: boolean): void
    {
        this.concluida = concluida;
    }
}

File: backend/springboot/src/main/java/com/todolist/models/Task.java
package com.todolist.models;

public class Task{
	private String nome;

	private Boolean concluida;

	public Task(String nome) {
		this.nome = nome;
		this.concluida = false;
	}

	public String getNome() {
		return nome;
	}

	public void setNome(String nome) {
		this.nome = nome;
	}

	public Boolean getConcluida() {
		return concluida;
	}

	public void setConcluida(Boolean concluida) {
		this.concluida = concluida;
	}
}
